CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# CODEOWNERS for autoreview assigning in github
+
+# https://help.github.com/en/articles/about-code-owners#codeowners-syntax
+# Order is important; the last matching pattern takes the most
+# precedence.
+
+# Root folder
+*      @krish2718 @sr1dh48r @rlubos @sachinthegreen
```

Comment:
In this particular case it'd make sense to squash the fix - you don't introduce a separate feature with a commit, but rather fix the other one present in the PR.